### PR TITLE
Nirms Question and Nirms Number

### PR DESCRIPTION
**Pull request by mycp98**
We kept this link component because it's just so nice. Happy to remove

### CODE REVIEW

**Review by yohan-oc — DISMISSED** [review]

File: app/views/NirmsNumberView.scala.html [hunk 1]
```diff
@@ -14,36 +14,44 @@
...
 
-@layout(pageTitle = title(form, messages("nirmsNumber.title"))) {
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsNumber.title")),
+    showBackLink = true
+) {
+
 
-    @formHelper(action = routes.NirmsNumberController.onSubmit(mode)) {
+    @formHelper(action = routes.NirmsNumberController.onSubmit(mode), Symbol("autoComplete") -> "off") {
```
Comment (liandra-giacomine) [inline app/views/NirmsNumberView.scala.html:38]: What does setting autoComplete to off do? Could it be useful for accessibilty to have it turned on?

File: app/views/HasNirmsView.scala.html [hunk 2]
```diff
@@ -19,12 +19,23 @@
...
 
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(form, messages("hasNirms.title"))) {
+    <h1 class="govuk-heading-l">@messages("hasNirms.h1")</h1>
+
+    <p class="govuk-body">@messages("hasNirms.p1")</p>
+
+    <p class="govuk-body">
+        @Html(messages("hasNirms.p2", link(messages("hasNirms.p2.linkText"),
```
Comment (besscerule) [inline app/views/HasNirmsView.scala.html]: do we want to be using the link component? I think this was the one Nick specifically warned against

File: app/views/HasNirmsView.scala.html [hunk 3]
```diff
@@ -35,7 +46,7 @@
         @govukRadios(
             RadiosViewModel.yesNo(
                 field = form("value"),
-                legend = LegendViewModel(messages("hasNirms.heading")).asPageHeading()
+                legend = Legend(content = "hasNirms.h1", classes = "govuk-visually-hidden")
```
Comment (besscerule) [inline app/views/HasNirmsView.scala.html:51]: why the change?

File: app/views/NirmsNumberView.scala.html [hunk 4]
```diff
@@ -14,36 +14,44 @@
...
+    govukErrorSummary: GovukErrorSummary
 )
 
+
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("nirmsNumber.title"))) {
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsNumber.title")),
+    showBackLink = true
```
Comment (besscerule) [inline app/views/NirmsNumberView.scala.html:34]: this defaults to true so this line isn't needed

File: app/views/components/Link.scala.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+@*
```
Comment (besscerule) [inline app/views/components/Link.scala.html]: i would try not to use this

**Review by besscerule — CHANGES_REQUESTED** [review]

**Review by besscerule — APPROVED** [review]

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**